tests/numpy/numeric_test.py

```diff
@@ -912,7 +889,70 @@ def test_putmask(self, prob_size):
             ak.putmask(pda, pda > 5, akvalues)
             assert np.allclose(nda, pda.to_ndarray())
 
-            # finally try to raise errors
+            # multi-dimensional case
+
```

Comment:
Would you please put the multi-dimensional case in a separate test and use the pytest.skip feature?

Reply:
Done.

Reply:
Could you change the name from `test_putmask_putmask` to `test_putmask_multidim` or similar?

Reply:
This is extremely strange.  On my VM, it is and always was named test_putmask_multidim.  I'm going to have to do some digging.